wemake_python_styleguide/visitors/ast/builtins.py

```diff
@@ -197,9 +197,9 @@ def _check_is_magic(self, node: ast.Constant) -> None:
 
         if isinstance(node.value, int) and node.value <= self._non_magic_modulo:
             return
-
+        real_value = self._token_dict[node.lineno, node.col_offset].string
```

Comment:
This can also be extracted to some helper method.

```suggestion
        try:
            token = self._token_dict[node.lineno, node.col_offset]
        except KeyError:
            # For some reason, the token was not found
            real_value = str(node.value)
        else:
            real_value = token.string
```